bakerydemo/base/migrations/0017_standardpage.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-03 08:07
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.wagtailcore.blocks
+import wagtail.wagtailcore.fields
+import wagtail.wagtailembeds.blocks
+import wagtail.wagtailimages.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0032_add_bulk_delete_page_permission'),
+        ('wagtailimages', '0019_delete_filter'),
```

Comment:
@shacker I did use a clean instance with no DB. This line is the offender; the reference to 0019 leads me to believe that this migration was created in a virtualenv that is using Wagtail master branch, as the 1.9 branch does not have this migration.